app/src/main/java/com/example/partyPlayer/MainActivity.kt

```diff
@@ -1,4 +1,4 @@
-package com.example.party_player
+package com.example.partyPlayer
```

Comment:
Changed the package names by removing underscores to comply with Kotlin naming conventions stated here: https://kotlinlang.org/docs/reference/coding-conventions.html